core/src/main/java/com/linecorp/armeria/server/DecodedHttpRequest.java

```diff
@@ -88,6 +88,8 @@ static DecodedHttpRequest of(boolean endOfStream, EventLoop eventLoop, int id, i
 
     void close(Throwable cause);
 
+    boolean isNormallyClosed();
```

Comment:
I know what normally means, but it doesn't really hit me. How about?
```suggestion
    boolean isClosedSuccessfully();
```

Reply:
That's a good name. 👍